test/integration/parameter-types/01-install.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: kudo.dev/v1beta1
+kind: Operator
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: parameter-test
+---
+apiVersion: kudo.dev/v1beta1
+kind: OperatorVersion
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: parameter-test-v1
+spec:
+  version: "1.0.0"
+  connectionString: ""
+  operator:
+    name: parameter-test
+    kind: Operator
+  templates:
+    deployment.yaml: |
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: nginx-deployment
+        labels:
+        {{ range $key, $value := .Params.LABELS }}
+          {{ $key }}: {{ $value }}
+        {{ end }}
+      spec:
+        selector:
+          matchLabels:
+            app: nginx
+        replicas: 1
+        template:
+          metadata:
+            labels:
+              app: nginx
+          spec:
+            containers:
+            - name: nginx
+              image: nginx:1.7.9
+              ports:
+              {{ range .Params.PORTS }}
+              - containerPort: {{ . }}
+              {{ end }}
+  tasks:
+    - name: operator
+      kind: Apply
+      spec:
+        resources:
+          - deployment.yaml
+  parameters:
+    - name: LABELS
+      description: "Deployment labels"
+      default: "a: a"
```

Comment:
I'm a bit lost in this PR, but AFAIR you said that while putting `interface{}` values into the CRD is not supported by `controller-gen`, we might be able to support non-strings on the UI side? Yet here I'm seeing strings only...

Reply:
That's because this is not an operator package but installing CR's directly. Take a look at the `parameter-types-operator`, where YAML is used in `params.yaml`.

Reply:
That's because this test directly writes CRDs and does not use the KUDO cli

Reply:
:man_facepalming: